packages/server/server/schema/types/root_query_type.js

```diff
@@ -83,7 +83,8 @@ const RootQueryType = new GraphQLObjectType({
           const coffees = await Coffee.find({
             "$or": [
               { "origin": { '$regex': filter, '$options': 'i' } },
-              { "name": { '$regex': filter, '$options': 'i' } }
+              { "name": { '$regex': filter, '$options': 'i' } },
+              { "roast": { '$regex': filter, '$options': 'i' } }
```

Comment:
This in not-intuitive, but the way this is being stored in our database is "roasting" so `roast` on line 87 actually needs to be `roasting`. I'll make this change real quick so you don't have to though because this is a weird little quirk of our app.